utils/src/stages/payout_stage.ts

```diff
@@ -26,11 +116,190 @@ export function createPayoutStage(
   config: Partial<PayoutStageConfig> = {}
 ): PayoutStageConfig {
   return {
-    id: generateId(),
+    id: config.id ?? generateId(),
     kind: StageKind.PAYOUT,
     game: config.game ?? StageGame.NONE,
     name: config.name ?? 'Payout',
     descriptions: config.descriptions ?? createStageTextConfig(),
     progress: config.progress ?? createStageProgressConfig(),
+    currency: config.currency ?? PayoutCurrency.USD,
+    payoutItems: config.payoutItems ?? [],
   };
 }
+
+/** Create default payout item. */
+export function createDefaultPayoutItem(
+  config: Partial<DefaultPayoutItem> = {},
+): DefaultPayoutItem {
+  return {
+    id: config.id ?? generateId(),
+    type: PayoutItemType.DEFAULT,
+    name: config.name ?? '',
+    description: config.description ?? '',
+    isActive: config.isActive ?? true,
```

Comment:
In what scenario would a payout item be created but not active?

Reply:
Right now, for LAS either Part 2 XOR Part 3 is not active. This was the least disruptive way I could think of to allow LAS experimenters to quickly "randomize" from the payout editor (the LAS game config also randomly selects the isActive on creation). We can take it out if you think it's okay to randomize only on the backend (e.g., only add one of the two payout items)?